### ISSUE TITLE
Could you share distill log for each dataset ? 

### ISSUE THREAD

**Issue by JarveeLee** [pengsongyou/openscene#30]
As title, for nuScenes, ScanNet, Matterport, it would be great if you can share the logs of distilling, which can help me to debug the training process ...

**Comment by pengsongyou** [comment]
Sorry I did not take the logs out from my internship, so I don't really have it anymore. Maybe you can specify what kind of issues you are encountering?

**Comment by JarveeLee** [comment]
I am distilling on my own data.
Occasionally I get 'WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.
' then the loss is Nan as well. Restart then NaN would disappear ... 

I have no idea how to explore this bug ..... 
![image](https://github.com/pengsongyou/openscene/assets/21051247/39d08935-d716-4636-b460-bac19a23e3b8)

**Comment by pengsongyou** [comment]
Emmmmm, I never encountered such issue before. Make sure that your data
preprocessing is correct, and the codes you changed do not divide by 0 or
so.

**Comment by wengmingao** [comment]
I meet the same problem too when I run distill on Matterport or ScanNet datasets.

NaN or Inf found in input 

![2023-07-06_193228](https://github.com/pengsongyou/openscene/assets/8683628/5f7dc521-baf6-474d-a8a8-c39b7790eff3)

Could anyone have some solution?

I try to add gradient clip to max 1 or lower the lr. But still no help.

What's more, I found it will be overfitting and val loss will be NaN. But the training process will not be interrupted.
![2023-07-06_193457](https://github.com/pengsongyou/openscene/assets/8683628/804dbe02-97ac-4399-bc9e-c5a944db887c)

I run code on Pytorch 2.0 with 4090 GPU. Maybe the Pytorch upgrade caused this problem?

**Comment by pengsongyou** [comment]
I will investigate this from my end and share the information to you later.
